Packs/SpaceX/Integrations/SpaceX/SpaceX.yml

```diff
@@ -0,0 +1,2912 @@
+category: Analytics & SIEM
+commonfields:
+  id: Space-X
+  version: -1
+configuration:
+- additionalinfo: This does not really need to be changed.
+  defaultvalue: https://api.spacexdata.com/v3
+  display: Server URL (e.g. https://api.spacexdata.com/v3)
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
```

Comment:
Add the `first_fetch` param to indicate on the initial launch to fetch (by flight id if I'm not mistaken)